File: app/src/main/java/ru/bk/klim9/movies/database/MoviesDatabase.java
package ru.bk.klim9.movies.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import ru.gdg.arturvasilov.popularmovies.data.database.dao.MoviesDao;
import ru.gdg.arturvasilov.popularmovies.data.movies.Movie;

/**
 * @author Artur Vasilov
 */
@Database(entities = {Movie.class}, version = 1)
public abstract class MoviesDatabase extends RoomDatabase {
    public abstract MoviesDao moviesDao();
}

File: app/src/main/java/ru/bk/klim9/movies/database/MoviesDatabase.kt
package ru.bk.klim9.movies.database

import androidx.room.Database
import androidx.room.RoomDatabase

import ru.bk.klim9.movies.movies.Movie

/**
 * @author ivan.a.klymenko@gmail.com on 2019-12-08
 */
@Database(entities = [Movie::class], version = 1, exportSchema = false)
abstract class MoviesDatabase : RoomDatabase() {
    abstract fun moviesDao(): MoviesDao
}
